eng/pipelines/ci-template.yml

```diff
@@ -118,7 +118,10 @@ steps:
       workingDirectory: $(Build.SourcesDirectory)/autorest.python/packages/${{parameters.folderName}}/
       condition: and(succeeded(), ${{ parameters.regenerate }}, eq('${{parameters.folderName}}', 'autorest.python'))
 
-    - script: npm run regenerate
+    - script: |
+        find test/azure/generated -type f ! \( -path "test/azure/generated/resiliency-srv-driven1/*" -o -name "*apiview_mapping_python.json*" \) -delete
+        find test/unbranded/generated -type f ! \( -path "test/unbranded/generated/resiliency-srv-driven1/*" -o -name "*apiview_mapping_python.json*" \) -delete
+        npm run regenerate
```

Comment:
- Keep `apiview_mapping_python.json` since not all environments generate it
- Keep `resiliency-srv-driven1` since it acts as legacy code and will not be generated